README.md

```diff
@@ -22,7 +22,7 @@
 
 ### Run a docker container from that image
 
-``` docker run -d -p 8081:8081 --name nexus-repository-apt:3.11.0 ```
+``` docker run -d -p 8081:8081 --name nexus-server nexus-repository-apt:3.11.0 ```
```

Comment:
BIKE SHEDDING: Might be worth naming it nexus-repo, that's more consistent with branding etc...

Reply:
Will do.